### ISSUE TITLE
Calling kv::set will freeze the HTTP handler

### ISSUE THREAD

**Issue by Mossaka** [deislabs/spiderlightning#208]
**Description of the bug**
That is, using kv capability inside of a http handler will render it frozen. 

**To Reproduce**
```rust
#[register_handler]
fn handle_bar(request: Request) -> Result<Response, Error> {
    assert_eq!(request.method, Method::Put);
    if let Some(body) = request.body {
        let kv = crate::Kv::open("my-container").unwrap();
        kv.set("key", &body).unwrap();
    }
    Ok(Response {
        headers: Some(request.headers),
        body: None,
        status: 200,
    })
}
```

**Additional context**

**Comment by Mossaka** [comment]
The cause of the issue: the implementation of HTTP service uses tokio runtime, while az blob storage and aws dynamodb use `futures::executor` runtime.

**Comment by Mossaka** [comment]
Closed by #209
